src/parsing/types/nep413/payload.rs

```diff
@@ -18,6 +18,7 @@ impl Payload {
             callback_url: Some(CappedString::new()),
         }
     }
+    /// must be only called once after `Self::new` to avoid unexpected panics
```

Comment:
Is there a way to avoid unexpected panics or forbid subsequent calls through ownership interface?

Reply:
https://github.com/dj8yfo/app-near-rs/pull/46#discussion_r1519872552